tests/suites/test_suite_pk.function

```diff
@@ -1440,43 +1458,42 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_PSA_CRYPTO_C:MBEDTLS_GENPRIME */
-void pk_psa_sign_ext(int pk_type, int parameter, int key_pk_type, int md_alg)
+/* BEGIN_CASE depends_on:MBEDTLS_GENPRIME */
+void pk_sign_ext(int pk_type, int parameter, int key_pk_type, int md_alg)
```

Comment:
Pre-existing: `parameter` is really not an informative name. Could you change that to a better name, like `key_bits`?

Reply:
Ah, good point, I missed that. Then perhaps `curve_or_bitsize`? Or if you're not convinced, let's just ignore it as it's preexisting.

Reply:
Went for `curve_or_keybits`.